extensions/amp-list/0.1/amp-list.js

```diff
@@ -72,6 +73,10 @@ export class AmpList extends AMP.BaseElement {
 
     /** @const @private {string} */
     this.initialSrc_ = element.getAttribute('src');
+
+    this.registerAction('refresh', () => {
+      this.fetchList_();
+    }, ActionTrust.LOW);
```

Comment:
Curious why low trust? Do you need this to trigger on scroll?

Reply:
No, I don't need it to trigger on scroll. I just figured there wasn't any reason to lock it down too tightly since its not really dangerous (like a form submit or something). I can change it to `ActionTrust.HIGH` if you want though. It doesn't really matter to me.

Reply:
Ok. I changed it to `ActionTrust.HIGH` now.